docs/basics/userguide/seeding.md

```diff
@@ -320,8 +351,11 @@ random sample 2 from seeder2 rng:  -0.1930990650226178
 </br>
 
 ## External libraries
-You can also use a seeder to seed external libraries (such as torch) using the method `set_external_seed`.
-It will be usefull if you want reproducibility with external libraries. In this example, you will use `torch` to generate random numbers.
+You can also use a seeder to seed some external libraries using the method `set_external_seed`.
```

Comment:
It is not really the seeder, it is more a tool in rlberry to properly seed external libraries (e.g. torch)